src/childinfo.c

```diff
@@ -64,7 +65,7 @@ void closeChildInfoPipe(void) {
 }
 
 /* Send save data to parent. */
-void sendChildInfoGeneric(childInfoType info_type, size_t keys, double progress, char *pname) {
+void sendChildInfoGeneric(childInfoType info_type, size_t keys, size_t output, double progress, char *pname) {
```

Comment:
```suggestion
void sendChildInfoGeneric(childInfoType info_type, size_t keys, size_t repl_output_bytes, double progress, char *pname) {
```

WDYT? I feel like it probably makes sense to just give it a dedicated field for now (as I mentioned - this child process info function needs some general refactoring)

Reply:
sure. I don't have a strong opinion about this, dedicated field kind of takes away the scope for overloading in the future, but I guess it can be refactored accordingly. I'll make this change

Reply:
Made the change and build is green :D